sqlmesh/core/test/definition.py

```diff
@@ -129,10 +129,9 @@ def setUp(self) -> None:
                         known_columns_to_types[col] = v_type
 
             test_fixture_table = self._test_fixture_table(name)
-            if test_fixture_table.db:
-                self.engine_adapter.create_schema(
-                    schema_(test_fixture_table.args["db"], test_fixture_table.args.get("catalog"))
-                )
+            self.engine_adapter.create_schema(
```

Comment:
Do we clean it up?

Reply:
Good catch, thanks! I intended to do this but forgot.

Reply:
I assumed `drop_schema` with `cascade=True` can take care of this - see: https://github.com/TobikoData/sqlmesh/pull/2461/commits/7c29450f61a4274a023156acddd970217b54bbf6.